docs/source/top_level_elements/location.rst

```diff
@@ -0,0 +1,208 @@
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2017033007">Frank H. McClung Museum of Natural History and Culture</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2017033007> .
+
+physicalLocation as string (special collections)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/roth:4245/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee, Knoxville. Special Collections</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation as string (libraries)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**The University of Tennessee Libraries, Knoxville** 574 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/fbpro:94819/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>The University of Tennessee Libraries, Knoxville</physicalLocation>
+    </location>
+
+**University of Tennessee Knoxville. Libraries** 664 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/tatum:609/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee Knoxville. Libraries</physicalLocation>
+    </location>
+
+**University of Tennesse Knoxville. Libraries** 397 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/tdh:8781/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennesse Knoxville. Libraries</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/n80003889> .
+
+physicalLocation and shelfLocator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/scopes:1258/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2014027633">University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <shelfLocator>Box 5, Folder 8</shelfLocator>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2014027633> ;
+        opaque:locationShelfLocator "Box 5, Folder 8" .
```

Comment:
URI resolves as:

"Hmmmm...

The vocabulary or term you were looking for doesn't exist."

Reply:
See https://github.com/UTKcataloging/mods_to_rdf/pull/29#discussion_r496734929